src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntityDueDiligenceValidatorTest.java

```diff
@@ -27,15 +27,24 @@ void init() {
         addressDtoValidator = new AddressDtoValidator();
         overseasEntityDueDiligenceValidator = new OverseasEntityDueDiligenceValidator(addressDtoValidator);
         overseasEntityDueDiligenceDto = OverseasEntityDueDiligenceMock.getOverseasEntityDueDiligenceDto();
+        overseasEntityDueDiligenceDto.getAddress().setCountry("England");
     }
 
     @Test
-    void testErrorReportedWhenIdentityDateFieldIsNow() {
+    void testNoErrorReportedWhenIdentityDateFieldIsNow() {
         overseasEntityDueDiligenceDto.setIdentityDate(LocalDate.now());
         Errors errors = overseasEntityDueDiligenceValidator.validate(overseasEntityDueDiligenceDto, new Errors(), CONTEXT);
         assertFalse(errors.hasErrors());
     }
 
+    @Test
```

Comment:
How about a test when the country is not in the Uk? An error should be reported. I think @markpit alluded to this in one of his comments.